node-tests/acceptance/exam-iterate-test.js

```diff
@@ -1,20 +1,24 @@
 'use strict';
 
 const assert = require('assert');
-const execa = require('execa');
-const rimraf = require('rimraf');
+const { rimrafSync } = require('rimraf');
 const fs = require('fs-extra');
 const path = require('path');
 
 function assertExpectRejection() {
   assert.ok(false, 'Expected promise to reject, but it fullfilled');
 }
 
+async function execa(command, args) {
+  const { execa: originalExeca } = await import('execa');
+  return originalExeca(command, args);
+}
+
 describe('Acceptance | Exam Iterate Command', function () {
   this.timeout(300000);
 
   it('should build the app, test it a number of times, and clean it up', function () {
```

Comment:
Alternatively, here, you could pass an async function as the second arg to `it` and use `await`